src/photos.cs

```diff
@@ -1005,9 +1005,15 @@ interface PHPhotoLibrary {
 
 	[iOS (9,1)]
 	[TV (10,0)]
+#if MONOMAC
+	[DisableDefaultCtor] // NS_UNAVAILABLE
```

Comment:
Is that for macOS only ? if not remove the define and we'll fix it when we look for API breakage